### PR TITLE
python.d(smartd_log): collect attribute 249 -- NAND Writes 1GiB

### PR DESCRIPTION
**Pull request by RaitoBezarius**
##### Summary

This enables the collection of NAND Writes 1GiB attribute, this enable measurement of write amplification phenomenon: https://en.wikipedia.org/wiki/Write_amplification

##### Component Name

python.d/smartd_log collector

##### Test Plan

Tested on a `SAMSUNG MZ7LN256HMJP-00000`, I suppose that consumer grade SATA SSD might expose a similar attribute compared to a real host writes metric. Knowing the WA, it's possible to guess the host writes metric, which is an important metric knowing the endurance of disks.

---

On another note, I would like to add Total LBAs written/read, would that be a good idea? (see https://www.thomas-krenn.com/en/wiki/SMART_attributes_of_Intel_SSDs)

### CODE REVIEW

File: collectors/python.d.plugin/smartd_log/smartd_log.chart.py [hunk 1]
```diff
@@ -330,7 +331,13 @@
         'lines': [],
         'attrs': [ATTR233],
         'algo': ABSOLUTE,
-    }
+    },
+    'nand_writes_1gib': {
+        'options': [None, 'NAND Writes 1GiB', 'blocks', 'wear', 'smartd_log.nand_writes_1gib', 'line'],
+        'lines': [],
+        'attrs': [ATTR249],
+        'algo': ABSOLUTE,
```
Comment (ilyam8) [inline collectors/python.d.plugin/smartd_log/smartd_log.chart.py:339]: from https://en.wikipedia.org/wiki/S.M.A.R.T.

> Total NAND Writes. Raw value reports the number of writes to NAND in 1 GB increments.[77]


Perhaps i am missing something, can't say i understand this attribute 😄 


- why units is `blocks`? 
- does absolute work? we are interested in total number of those _writes_ and not _writes_ rate (incremental)?

Comment (RaitoBezarius) [reply collectors/python.d.plugin/smartd_log/smartd_log.chart.py:339]: Hm, you're right! I was copying blindly what was up and was not sure if it was right.

**Comment by ilyam8** [comment]
> On another note, I would like to add Total LBAs written/read, would that be a good idea?

Looks good to me.

File: collectors/python.d.plugin/smartd_log/smartd_log.chart.py [hunk 2]
```diff
@@ -330,7 +331,13 @@
         'lines': [],
         'attrs': [ATTR233],
         'algo': ABSOLUTE,
-    }
+    },
+    'nand_writes_1gib': {
+        'options': [None, 'NAND Writes 1GiB', 'blocks', 'wear', 'smartd_log.nand_writes_1gib', 'line'],
```
Comment (ilyam8) [inline collectors/python.d.plugin/smartd_log/smartd_log.chart.py:336]:
```suggestion
        'options': [None, 'NAND Writes (1GiB)', 'GiB', 'wear', 'smartd_log.nand_writes_1gib', 'line'],
```

**Review by ilyam8 — CHANGES_REQUESTED** [review]

File: collectors/python.d.plugin/smartd_log/smartd_log.chart.py (hunk 2 above)
Comment (ilyam8) [reply collectors/python.d.plugin/smartd_log/smartd_log.chart.py:336]: @RaitoBezarius i suggest to change units to GiB. Apart from that lgtm.

**Review by ilyam8 — APPROVED** [review]

**Review by thiagoftsm — APPROVED** [review]

**Review by vlvkobal — APPROVED** [review]

**Comment by ilyam8** [comment]
Thanks for the contribution @RaitoBezarius !

**Pull request merged**